Shared/Features/Projects/View/ProjectsView.swift

```diff
@@ -1,8 +1,28 @@
 import SwiftUI
 
 struct ProjectsView: View {
+    init() {
+        let coloredNavAppearance = UINavigationBarAppearance()
+        coloredNavAppearance.titleTextAttributes = [.foregroundColor: UIColor(Color("SecondaryColor"))]
+        coloredNavAppearance.backgroundColor = UIColor(Color("AccentColor"))
+
```

Comment:
This nav setup is usually done once since `UINavigationBar.appearance()` applies for every instance of the UINavigationBar. If we already have this on the top UI component, it won't be necessary to do it again.